hoodie-common/src/main/java/com/uber/hoodie/common/table/log/HoodieLogFileReader.java

```diff
@@ -125,6 +128,7 @@ private HoodieLogBlock readBlock() throws IOException {
 
         // 2 Read the total size of the block
         blocksize = inputStream.readInt();
+        this.logBlockSize = blocksize;
```

Comment:
Not related to this change but important to fix: In line 137, We are doing catch-all exceptions and treating as corrupt block. The underlying inputStream could throw IOException (other than EOF exception) because of any transient failures (or like stream getting closed) and these doesn't mean it was because of corrupt block. These IOException cases needs to raised to the caller instead of returning a corrupt block.

Reply:
Good point, fixed it.